src/actions/history.js

```diff
@@ -131,11 +133,12 @@ function loadNewer(params) {
     // Ensures we don't have useless requests to the backend.
     if (newerMessagesRequest) return
 
-    const promise = api.loadHistory(channel.id, {
-      limit: stopIndex - startIndex,
-      timeFrom: last(messages).time,
-      sort: 'time:asc',
-    })
+    const promise = api.loadNewerHistory(
+      channel.id,
+      stopIndex - startIndex,
```

Comment:
@nikgraf Don't you want to combine last 3 params in one object, like `params` or `options`?

Reply:
I was thinking about it, but on the other hand I wanted to indicate that alle 4 parameters are mandatory. I suggest if we convert it to an object we should accept all 4 inside one object since all are required and should be treated equally.

What do you think? Should we switch to an object.